### PR TITLE
Demonstrate how to consume an award update presented as serialised JSON

### PR DESCRIPTION
**Pull request by shtukas**
AWS test function sends

```
"{\"category\": \"Best Actor\", \"winner\": \"Pascal\"}"
```

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -37,13 +37,20 @@ addCommandAlias("dist", ";riffRaffArtifact")
 resolvers += "Sonatype releases" at "https://oss.sonatype.org/content/repositories/releases"
 
 val AwsSdkVersion = "1.10.74"
+val CirceVersion = "0.7.0-M1"
```
Comment (tomrf1) [inline build.sbt]: can use 0.7.0 now

File: src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import com.amazonaws.services.lambda.runtime.{ Context, RequestHandler }
+import com.amazonaws.services.lambda.runtime.events.SNSEvent
+import io.circe.{ Json, ParsingFailure }
+
+import scala.collection.JavaConverters._
+import io.circe.generic.auto._
+import io.circe.parser._
+import io.circe.syntax._
+import io.circe.jawn.JawnParser
```
Comment (tomrf1) [inline src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala]: there must be at least 1 unnecessary import here

**Comment by tomrf1** [comment]
👍 besides comments

File: src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala (hunk 2 above)
Comment (shtukas) [inline src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala]: Oh yes, sorry. Removing!

File: build.sbt (hunk 1 above)
Comment (shtukas) [inline build.sbt]: Done!

File: src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala (hunk 2 above)
Comment (shtukas) [inline src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala]: Done!

**Comment by shtukas** [comment]
Updated and tested. Merging

**Comment by LATaylor-guardian** [comment]
Can we please ensure Team City is correctly setup to report the build to status to Github. Shout if you need help doing so.

File: src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+import com.amazonaws.services.lambda.runtime.{ Context, RequestHandler }
+import io.circe.generic.auto._
+import io.circe.parser.decode
+
+case class AwardEvent(category: String, winner: String)
+
+class Lambda extends RequestHandler[String, Unit] {
+
+  def decodeInstruction(input: String): Option[AwardEvent] = {
```
Comment (LATaylor-guardian) [inline src/main/scala/com/gu/facebook-news-bot-oscars-notifications/Lambda.scala]: This can be private.

**Pull request merged**